### DISCUSSION TITLE
Credits in movies

### DISCUSSION THREAD

**Discussion by falumas** [time4tea/gopro-dashboard-overlay#90] [General]
I usually thank people who should have credit in my movies. And this software is well worthy of thanks.

Would you like to be thanked in movies, if yes with your name or time4tea or something else?

**Comment by time4tea** [comment]
Hi -Thank you! -  Well - a link to https://github.com/time4tea/gopro-dashboard-overlay/ would be very much appreciated...
Thanks! James

**Reply by time4tea** [reply]
Oh.. and please do post a link to the movie here if you can! It's great to see what people have created!
